google-cloud-monitoring/Rakefile

```diff
@@ -21,17 +19,53 @@ RuboCop::RakeTask.new
 
 desc "Run tests."
 task :test do
-  puts "The google-cloud-monitoring gem does not have tests."
+  $LOAD_PATH.unshift "lib", "test"
+  Dir.glob("test/**/*test.rb")
+    .reject { |file| file.include? "smoke_test" }
+    .each { |file| require_relative file }
 end
 
 namespace :test do
-  desc "Run tests with coverage."
+  desc "Runs tests with coverage."
+  task :coverage do
+    require "simplecov"
+    SimpleCov.start do
+      command_name "google-cloud-monitoring"
+      track_files "lib/**/*.rb"
+      add_filter "test/"
+    end
+
+    Rake::Task["test"].invoke
+  end
+end
+
+desc "Runs the smoke tests."
+task :smoke_test do
```

Comment:
Should `smoke_test` have been renamed `acceptance` here (monitoring)?

Reply:
The `smoke_test`s (which should start being added soon...) are a subset of `acceptance` tests, so I'm keeping the tasks separate. For GAPIC-only releases, the smoke test may be the only acceptance test, but for others, we might want a way to invoke manual system tests separately from autogen smoke tests.

Reply:
I do worry about the smoke tests being in the `test` directory, because they make actual calls to the API. I worry that they may be invoked during the build, like during the coverage task which loads all files under `test`.

Reply:
Acknowledged, but can we defer this discussion to when the smoke tests are added? No such smoke tests are included in this PR.

Reply:
Yep, I'm not intending to request any changes here, just bringing it up.